packages/page-bounties/src/BountyCreate.tsx

```diff
@@ -0,0 +1,91 @@
+// Copyright 2017-2020 @polkadot/app-bounties authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import BN from 'bn.js';
+import React, { useState } from 'react';
+
+import { useTranslation } from '@polkadot/app-accounts/translate';
+import { calculateBountyBond } from '@polkadot/app-bounties/helpers/calculateBountyBond';
+import { Button, Input, InputAddress, InputBalance, Modal, TxButton } from '@polkadot/react-components';
+import { useApi, useToggle } from '@polkadot/react-hooks';
+import { BN_ZERO } from '@polkadot/util';
+
+function BountyCreate () {
+  const { t } = useTranslation();
+  const { api } = useApi();
+
+  const bountyDepositBase = api.consts.treasury.bountyDepositBase;
+  const bountyDepositPerByte = api.consts.treasury.dataDepositPerByte;
+
+  const [accountId, setAccountId] = useState<string | null>(null);
+  const [title, setTitle] = useState('');
+  const [bond, setBond] = useState(bountyDepositBase.toBn());
+  const [amount, setAmount] = useState<BN | undefined>(BN_ZERO);
+  const [isValid] = useState(true);
+  const [isOpen, toggleIsOpen] = useToggle();
+
+  const onTitleChange = (value: string) => {
```

Comment:
useCallback rather.

Reply:
... which would probably meant you don't want to pass the consts `bountyDepositBase`, `bountyDepositPerByte` in, but rather the API. The former will change on each render, the later won't.